src/IntersectionObserver.svelte

```diff
@@ -16,22 +17,32 @@
   /** @type {null | Entry} */
   export let entry = null;
   export let intersecting = false;
+  /** @type {null | Observer} */
+  export let observer = undefined;
```

Comment:
```suggestion
  export let observer = null;
```

To be consistent with the JSDoc type.